src/Auxiliary/Aux_Check_Parameter.cpp

```diff
@@ -1108,8 +1135,69 @@ void Aux_Check_Parameter()
       Aux_Error( ERROR_INFO, "ELBDM_BASE_SPECTRAL must work with SUPPORT_FFTW !!\n" );
 #  endif
 
-#  if ( WAVE_SCHEME == WAVE_FD )
+// check hybrid scheme parameters for errors
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+
+   if ( OPT__INIT == INIT_BY_FILE && OPT__UM_IC_LEVEL >= ELBDM_FIRST_WAVE_LEVEL && ELBDM_MATCH_PHASE )
+      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support loading both fluid and wave levels from UM_IC because of phase matching !!\n" );
+
+   if ( INIT_SUBSAMPLING_NCELL > 1 )
+      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support INIT_SUBSAMPLING_NCELL > 1 !!\n" );
+
+   if ( OPT__OUTPUT_TOTAL == 2 )
+      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support OPT__OUTPUT_TOTAL == 2 !!\n" );
+
+#  if ( FLU_GHOST_SIZE < HYB_GHOST_SIZE )
+#  error : ERROR : FLU_GHOST_SIZE needs to be bigger than HYB_GHOST_SIZE !!
```

Comment:
This combination of compilation options will trigger this error if we don't change the default value of `FLU_GHOST_SIZE`.
```
--elbdm_scheme=ELBDM_HYBRID 
--wave_scheme=WAVE_FD
--laplacian_four=false 
```
I am not sure it is expected, and maybe it can be easily avoided by choosing a larger `FLU_GHOST_SIZE`.

Reply:
@JeiwHuang 
Thank you for pointing that out. I made the error message more descriptive for now. I also think it can be easily avoided by choosing a larger `FLU_GHOST_SIZE`. However, I was not sure whether that could have unintended side effects. 
And to be honest, I did not want to spend time testing it. 

@hyschive 
Do you know whether there could be unintended side effects? If you like, I can just change the ghost size when the hybrid scheme is on and test whether it compiles properly and a simple test case works fine. In that case, I would remove the error message.

Reply:
@KunkelAlexander 
>If you like, I can just change the ghost size when the hybrid scheme is on and test whether it compiles properly and a simple test case works fine. In that case, I would remove the error message.

Let's do that. An example code is given below. I believe that the only side effect is that it will affect the performance of the finite-difference solver and parallel efficiency, which is not a big concern as we should enable `LAPLACIAN_4TH` by default. Also, I've confirmed that for the `GaussianWavePacket` test, `FLU_GHOST_SIZE = 3` and `6` give identical results with `LAPLACIAN_4TH` disabled.

```
#     ifdef LAPLACIAN_4TH
#        define FLU_GHOST_SIZE         6
#     else
//     hybrid scheme requires FLU_GHOST_SIZE >= HYB_GHOST_SIZE (6)
#      if ( ELBDM_SCHEME == ELBDM_HYBRID )
#        define FLU_GHOST_SIZE         6
#      else
#        define FLU_GHOST_SIZE         3
#      endif
#     endif
```

Reply:
@hyschive
Excellent, Thank you for your reply. I modified the code accordingly and will test it as well.